src/validate.js

```diff
@@ -52,12 +52,24 @@ module.exports = validators => {
       integer: (value, min, max) => {
         min = is.defined(min) ? min : -Infinity;
         max = is.defined(max) ? max : Infinity;
-        return is.integer(value) && value >= min && value <= max;
+        if (!is.integer(value)) {
+          return false;
+        }
+        if (value < min || value > max) {
+          throw new Error(`Value must be within [${min}, ${max}] inclusive.`);
```

Comment:
Optional: add `value` to the error message? (for example, append `, but was ${value}.`.